src/map/entities/battleentity.cpp

```diff
@@ -111,6 +111,15 @@ bool CBattleEntity::isInDynamis()
     return false;
 }
 
+bool CBattleEntity::isInAssault()
+{
+    if (loc.zone != nullptr) 
+    {
+        return loc.zone->GetType() == ZONETYPE_DUNGEON_INSTANCED && (loc.zone->GetRegionID() >= 28 && loc.zone->GetRegionID() <= 32);
```

Comment:
region enums

Reply:
Sorry dont know what you mean

Reply:
return loc.zone->GetType() == ZONETYPE_DUNGEON_INSTANCED && (loc.zone->GetRegionID() >= 28 && loc.zone->GetRegionID() <= 32);

to

 return loc.zone->GetType() == ZONETYPE_DUNGEON_INSTANCED && (loc.zone->GetRegionID() >= REGION_WEST_AHT_URHGAN && loc.zone->GetRegionID() <= REGION_ALZADAAL);

is that what your looking for?